.nycrc

```diff
@@ -0,0 +1,11 @@
+{
+  "include": [
+    "**/**/src/*.ts"
+  ],
+  "exclude": [
+    "**/**/**/*.spec.js"
```

Comment:
Do we need this exclude block if this doesnt actually get covered by anything in the include section?